lib/kitchen/driver/ec2.rb

```diff
@@ -427,7 +427,7 @@ def hostname(server, interface_type = nil)
 
       def create_ec2_json(state)
         if windows_os?
-          cmd = "mkdir \\etc\\chef\\ohai\\hints; echo $null >> \\etc\\chef\\ohai\\hints\\ec2.json"
+          cmd = "New-Item -Force C:\\chef\\ohai\\hints\\ec2.json -Type File"
```

Comment:
The Type alias (for ItemType) wasn't introduced until PowerShell 3.  Server 2008R2 ships with PS 2 by default and this will work fine if you change the parameter to -ItemType (which will be valid from PowerShell 1 and up)